bin/pipeline_result_processor/prp/cli.py

```diff
@@ -0,0 +1,154 @@
+import json
+import logging
+
+import click
+from pydantic import ValidationError
+
+from .models.metadata import RunInformation, SoupVersion
+from .models.phenotype import PhenotypeType
+from .models.sample import MethodIndex, PipelineResult
+from .parse import (
+    parse_cgmlst_results,
+    parse_mlst_results,
+    parse_qust_results,
+    parse_resistance_pred,
+    parse_species_pred,
+    parse_virulence_pred,
+)
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s] %(levelname)s in %(module)s: %(message)s"
+)
+LOG = logging.getLogger(__name__)
+
+OUTPUT_SCHEMA_VERSION = 1
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+@click.option("-i", "--sample-id", required=True, help="Sample identifier")
+@click.option(
+    "-u",
+    "--run-metadata",
+    type=click.File(),
+    required=True,
+    help="Analysis metadata from pipeline",
+)
+@click.option("-q", "--quast", type=click.File(), help="Quast quality control metrics")
+@click.option(
+    "-p",
+    "--process-metadata",
+    type=click.File(),
+    multiple=True,
+    help="Meta data from nextflow processes",
+)
+@click.option(
```

Comment:
Possibly clarify the help text more, these are inputs right? Maybe write what format the file should be in or which software that produce the file used as input